src/main/java/org/apache/cassandra/sidecar/MainModule.java

```diff
@@ -144,6 +148,22 @@ public Router vertxRouter(Vertx vertx, LoggerHandler loggerHandler, ErrorHandler
         // Docs index.html page
         StaticHandler docs = StaticHandler.create("docs");
         router.route().path("/docs/*").handler(docs);
+
+        // add custom routers
+        final String apiVersion = "/api/v1";
+        final String componentRoute = "/keyspace/:keyspace/table/:table/snapshots/:snapshot/component/:component";
+        final String defaultStreamRoute = apiVersion + componentRoute;
```

Comment:
so... the change drops "stream" in the endpoint?

Reply:
I think this reverts a breaking change that introduced the stream, and we wanted to fix this in order not to break the API in minor versions